tests/dslash_ctest.cpp

```diff
@@ -42,6 +42,10 @@ DiracDomainWall4DPC *dirac_4dpc = nullptr; // create the 4d preconditioned DWF D
 
 QudaDagType not_dagger;
 
+bool test_split_grid = false;
```

Comment:
There's (even more) boilerplate replication between `dslash_test` and `dslash_ctest` here.  Can we clean this up and move some of this into a common file, e.g., in `utils/`

Reply:
And the staggered flavors ;)

Reply:
I guess ... I can volunteer to do this? :)